packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -206,16 +206,16 @@ const helpers = {
   negotiateOptionsBySpecificCode(
     httpOperation: IHttpOperation,
     desiredOptions: NegotiationOptions,
-    code: string
+    code: number
   ): RE.ReaderEither<Logger, Error, IHttpNegotiationResult> {
     // find response by provided status code
     return pipe(
       withLogger(logger =>
         pipe(
-          findResponseByStatusCode(httpOperation.responses, code),
+          findResponseByStatusCode(httpOperation.responses, code.toString()),
           O.alt(() => {
             logger.info(`Unable to find a ${code} response definition`);
-            return createResponseFromDefault(httpOperation.responses, code);
+            return createResponseFromDefault(httpOperation.responses, code.toString());
```

Comment:
ditto

Reply:
Resolving this convo as the outcome of the convo [here](https://github.com/stoplightio/prism/pull/1542#discussion_r529062453) will apply here too.